MANIFEST.in

```diff
@@ -15,6 +15,7 @@ include LICENSE
 include MANIFEST.in
 include *.md
 include *.toml
+include azure-pipelines.yml
```

Comment:
That's not supposed to be included FYI (it's pruned at the end of the file).

Reply:
Thank you for the comment. I removed the line.